cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Query.hs

```diff
@@ -350,9 +351,15 @@ hprop_cli_queries = integrationWorkspace "cli-queries" $ \tempAbsBasePath' -> H.
           drepStateRedactedOutFile
           "test/cardano-testnet-test/files/golden/queries/drepStateOut.json"
 
-    TestQueryDRepStakeDistributionCmd ->
+    TestQueryDRepStakeDistributionCmd -> do
       -- drep-stake-distribution
-      H.noteM_ $ execCli' execConfig [ eraName, "query", "drep-stake-distribution", "--all-dreps" ]
+      -- to stdout
+      drepStakeDistribution :: [(L.DRep StandardCrypto, L.Coin)] <- H.noteShowM $ execCliStdoutToJson execConfig [ eraName, "query", "drep-stake-distribution", "--all-dreps" ]
+
+      -- TODO: we could check that the Coin amount below is the one reported
+      -- by query stake-address-info
+
+      forM_ drepStakeDistribution $ \(_drep, coin) -> coin H.=== Coin 300_000_000_000
```

Comment:
Why no list lenght check?
Also small nitpick, because how hedgehog renders failure, it's a little more readable to put expected value on the left hand side.
```suggestion
      forM_ drepStakeDistribution $ \(_drep, coin) -> Coin 300_000_000_000 H.=== coin
```

Reply:
@carbolymer> I added a list check and swapped the operands of `H.===` :+1: